tests/e2e/device-plugin_test.go

```diff
@@ -71,6 +85,13 @@ var _ = NVDescribe("GPU Device Plugin", func() {
 			},
 		}
 
+		// check Collector objects
+		if *CollectLogs == "default" {
+			collectLogsFrom = strings.Join(defaultCollectorObjects, ",")
+		} else {
+			collectLogsFrom = *CollectLogs
+		}
+
```

Comment:
if we change the parse function to accept a `...string` then we can rather do:
```suggestion
		collectLogsFrom = defaultCollectorObjects
		// check Collector objects
		if *CollectLogs != nil && *CollectLogs != "default" {
			collectLogsFrom = strings.Split(*CollectLogs, ",")
		}
```

Reply:
check